geode-core/src/main/java/org/apache/geode/internal/cache/AbstractRegionMap.java

```diff
@@ -388,9 +378,21 @@ public void close(BucketRegion bucketRegion) {
     if (logger.isDebugEnabled()) {
       logger.debug("Clearing entries for {} rvv={}", _getOwner(), rvv);
     }
-    LocalRegion lr = _getOwner();
+    final LocalRegion lr = _getOwner();
     RegionVersionVector localRvv = lr.getVersionVector();
-    incClearCount(lr);
+
+    boolean isBucketRegion = false;
+
+    if (!(lr instanceof HARegion)) {
```

Comment:
I don't think this code needs to check for HARegion. isUsedForPartitionedRegionBucket is all it needs to check for.

Reply:
removed the check